Source/Mosa.Compiler.Framework/Stages/BaseTransformStage.cs

```diff
@@ -39,13 +39,13 @@ public abstract class BaseTransformationStage : BaseMethodCompilerStage
 
 		protected BitArray EmptyBlocks;
 
-		protected Dictionary<string, Counter> transformationsCounts = new Dictionary<string, Counter>();
+		protected Dictionary<string, Counter> transformCounts = new Dictionary<string, Counter>();
 
 		protected bool CountTransformations = false;
 
-		public BaseTransformationStage(bool enableTransformationOptimizations = false, bool enableBlockOptimizations = false, int maxPasses = MaximumPasses)
+		public BaseTransformStage(bool enableTransformatOptimizations = false, bool enableBlockOptimizations = false, int maxPasses = MaximumPasses)
```

Comment:
I think `enableTransformatOptimizations` should have been `enableTransformOptimizations`